assignments/assignment02.rb

```diff
@@ -45,7 +77,9 @@ def median(ary)
 
 # implement method `pluck` on array of hashes
 def pluck(ary, key)
-  # your implementation here
+  plucked = []
```

Comment:
use map instead of iteration with each. then, you don't need to create your own array 'plucked', since map does that for you.